demo/index.html

```diff
@@ -29,12 +25,21 @@
 		</div>
 		<div class="vertical-section-container centered">
 			<h3>Button Dialog Multiple Roots Demo</h3>
-			<demo-snippet>
-				<template is="dom-bind">
-					<iron-ajax url="data/multiRootTree.json" handle-as="json" last-response="{{ treeData1 }}" auto></iron-ajax>
-					<cosmoz-treenode-button-view value="{{ path }}" chosen-node="{{ nodeChosen1 }}" data="{{ treeData1 }}"></cosmoz-treenode-button-view>
+			  <demo-snippet>
+				<template is="dom-bind" id="demo">
+					<iron-ajax url="data/multiRootTree.json" handle-as="json" on-response="handleResponse" debounce-duration="10" auto></iron-ajax>
+					<cosmoz-treenode-button-view id="navigator" value="{{ path }}" chosen-node="{{ nodeChosen1 }}"></cosmoz-treenode-button-view>
 				</template>
-			</demo-snippet>
-		</div>
+			  </demo-snippet>
+		 </div>
+		<script>
+			window.addEventListener('WebComponentsReady', function(e) {
+				document.querySelector('#demo').handleResponse = function(response) {
```

Comment:
This could actually be further simplified..
```html
<iron-ajax ... last-response="{{ treeData }}">
<cosmoz-treenode-button-view ... tree="[[ getTree(treeData) ]]">
```
```js
document.querySelector('#demo').getTree = function (data) {
  return new Cosmoz.DefaultTree(data);
};
```

Still, I don't think you need `window.addEventListener('WebComponentsReady', function(e) {` either?